g2p/tests/test_cli.py

```diff
@@ -166,9 +171,19 @@ def test_convert_option_t(self):
         self.assertIn("eːʔi", result.stdout)
 
     def test_convert_option_tl(self):
-        result = self.runner.invoke(convert, "--tok-lang fra e\\'i oji oji-ipa")
+        result = self.runner.invoke(
+            convert, "--tok-lang fra e\\'i oji oji-ipa")
         self.assertIn("eː'i", result.stdout)
 
+    def test_local_config(self):
+        config_path = os.path.join(PUBLIC_DIR, 'mappings', 'test.yaml')
+        result = self.runner.invoke(
+            convert, f"bbbb local-config-in local-config-out --config {config_path}")
+        self.assertIn("aaaa", result.stdout)
+        result = self.runner.invoke(
+            convert, f"b local-config-in eng-ipa --config {config_path}")
+        self.assertIn("ɑ", result.stdout)
+
```

Comment:
You cannot use literal `/` as path joiner, you have to use os.path.join. On my Windows machine:
```
$ ./test_cli.py
INFO - Server initialized for eventlet.
INFO - Running 355 g2p convert test cases found in public/data
.......FFE.
======================================================================
ERROR: test_scan_str_case (__main__.CliTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_cli.py", line 140, in test_scan_str_case
    returned_set = re.search('{(.*)}', result.stdout).group(1)
AttributeError: 'NoneType' object has no attribute 'group'

======================================================================
FAIL: test_local_config (__main__.CliTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_cli.py", line 182, in test_local_config
    self.assertIn("aaaa", result.stdout)
AssertionError: 'aaaa' not found in "Usage: convert [OPTIONS] INPUT_TEXT IN_LANG OUT_LANG\nTry 'convert -h' for help.
\n\nError: Invalid value for '--config': File 'c:usersjoaniserasg2pg2ptestspublicmappingstest.yaml' does not exist.\n
"

======================================================================
FAIL: test_scan_fra_simple (__main__.CliTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_cli.py", line 128, in test_scan_fra_simple
    self.assertEqual(result.exit_code, 0)
AssertionError: 2 != 0

----------------------------------------------------------------------
Ran 11 tests in 10.011s

FAILED (failures=2, errors=1)
```

Reply:
Hum, it fixes one instance, which is still showing as an error here, but 6684cbb79 replaces a bunch of `os.path.join()` by f-strings with hard-coded slashes, in particular in `not_test_scan_fra()`, `test_scan_fra_simple()` and `test_scan_str_case()`. I don't get why you would have made these changes, actually, maybe they're other auto edits?

But I wonder, you're using os.path.join to construct the file name, then building a command line from it. I normally don't build the command line as f-strings in my unit testing, but rather as an array of argument. This might make a difference on Windows it a way it might not be relevant on *nix.